src/main/java/Bob/Ui.java

```diff
@@ -0,0 +1,56 @@
+package bob;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Scanner;
+
+import bob.task.Task;
+
+/**
+ * Represents the CLI user interface of Bob, depreciated for the GUI. Will be removed in future versions.
+ */
+public class Ui {
+
+    /**
+     * Prints a dialogue.
+     *
+     * @param input
+     */
+    public static void printDialogueCLI(String input) {
```

Comment:
```suggestion
    public static void printDialogueCli(String input) {
```
Should be `Cli` instead of `CLI` following code conventions